src/components/Reviews/SingleReviewPage.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import * as api from "../../api";
+import { ReactComponent as CommentIcon } from "../../icons/comment.svg";
+import { ReactComponent as VotesIcon } from "../../icons/votes.svg";
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+dayjs.extend(relativeTime);
+
+const SingleReviewPage = () => {
+  const [isLoading, setLoading] = useState(true);
+  const [review, setReview] = useState([]);
+  const { review_id } = useParams();
+
+  useEffect(() => {
+    api.fetchSingleReview(review_id).then((review) => {
+      setReview(review);
+      setLoading(false);
```

Comment:
I can see you have implemented isLoading however when I slow down my connection I can't see any indication of this. Please could you try this your end to make sure its just a problem with my browser :)